cartridges/int_cloudinary_sfra/cartridge/client/default/js/components/miniCart.js

```diff
@@ -2,12 +2,18 @@
 
 module.exports = function () {
     // Custom Start: Make cart product images responsive //
```

Comment:
is it possible that the minicart would be called twice, and we will create duplicate observers?
Maybe it makes sense to use a global DOM variable to skip a second call to the minicart

Reply:
It only call once.